src/views/UserDetails.vue

```diff
@@ -272,8 +272,11 @@
                 <ion-icon :icon="addOutline" slot='start' />
                 {{ translate('Add to security group') }}
               </ion-button>
+              <ion-card-content v-if="!selectedUser.userLoginId">
+                {{ translate('Security groups can only be assigned after a login is created. Please add login credentials for above.') }}
+              </ion-card-content>
 
-              <ion-item>
+              <ion-item v-if="selectedUser.userLoginId && userGroupAssocHistories.length">
```

Comment:
Remove the check for user histories from the history button and also its relevant code.
Refer to the issue comments for clarity.